build/android/Makefile

```diff
@@ -15,6 +15,9 @@ ROOT = $(shell pwd)
 
 GAMES_TO_COPY = minetest_game
 
+# has to be the same value as the one you pass in
+PROJECT_NAME = minetest
```

Comment:
On other operating systems, we store our data to `.minetest` too.

Also, I don't change the capitalisation, I chose one. Before, we wrote to ``Minetest`, but read from `minetest`, which resulted in the problem that we only worked with case insensitive filesystems.